SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Supported Versions
+
+Patches for [ **illinois-app** ] will only be applied to the following versions:
+
+| Version | Supported |
+| ------- | ------------------ |
+| 3.0.18 | :white_check_mark: |
```

Comment:
Please add newer versions to this file in the develop branch. Thanks.

Reply:
Hi @sandeep-ps, thanks for reminding me. However I am not quite sure when and how I should update this file. I presume I should update it each time I create a release pull request to master, updating it in a way like [this](https://github.com/rokwire/illinois-app/commit/3e4f8d952ee499aed351bed6ca26e904cefc23e2)? Is that correct?